packages/pwa/package.json

```diff
@@ -112,8 +112,8 @@
         "analyze-build": "cross-env MOBIFY_ANALYZE=true npm run prod:build",
         "build-translations": "npm run extract-default-translations && npm run compile-translations",
         "compile-translations": "formatjs compile-folder --ast app/translations app/translations/compiled",
-        "compile-translations:pseudo": "formatjs compile --ast app/translations/$npm_package_l10n_defaultLocale.json --out-file app/translations/compiled/en-XB.json --pseudo-locale en-XB",
-        "extract-default-translations": "formatjs extract 'app/**/*.{js,jsx}' --out-file app/translations/$npm_package_l10n_defaultLocale.json --id-interpolation-pattern [sha512:contenthash:base64:6]",
+        "compile-translations:pseudo": "formatjs compile --ast app/translations/en-GB.json --out-file app/translations/compiled/en-XB.json --pseudo-locale en-XB",
+        "extract-default-translations": "formatjs extract 'app/**/*.{js,jsx}' --out-file app/translations/en-GB.json --id-interpolation-pattern [sha512:contenthash:base64:6]",
```

Comment:
```suggestion
        "compile-translations:pseudo": "formatjs compile --ast app/translations/en-US.json --out-file app/translations/compiled/en-XB.json --pseudo-locale en-XB",
        "extract-default-translations": "formatjs extract 'app/**/*.{js,jsx}' --out-file app/translations/en-US.json --id-interpolation-pattern [sha512:contenthash:base64:6]",
```

@alexvuong @bendvc We'll need to match this `react-intl`'s default locale with what's hardcoded in IntlProvider:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/2cf49564c888de6f67bf9b19a21b58642d60175a/packages/pwa/app/components/_app/index.jsx#L166-L170

Reminder that unlike the default locale for each DW site, this is the default locale for react-intl, and it's meant to be the only one for the entire project. Currently, our translations are in English. So we just need to pick either `en-US` or `en-GB`.

I understand that currently there's some difference between the pwa package and the generated project. So maybe that's where the confusion came from. 

Looks to me there are a few available options:
- We **hardcode the locale** and pick either `en-US` or `en-GB` ← the simplest option
- We **create new generator assets** for pwa/package.json and App component, to vary the locale accordingly ← I'm not sure this is a good idea, as having additional assets requires more maintenance.
- Instead of hardcoding the locale, we have it **be in our config file**

Reply:
I remember this. I feel like the easiest solution is best. As long as there are no side effects setting this default value with regards to the site. I vaguely remember that in the past because RefArch doesn't have en-GB and RefArch doesn't have en-US there was a big of an issue. I'm not sure if that limitation exists anymore. Just thought I would bring it up incase it still is.

Reply:
If I remember correctly, by default when RefArch is created in the sandbox, `en-GB` is not turned on, and vice versa for RefArchGlobal, which could return 500 to the Retail app if we make a call with a locale that has not turned on. 🤔 .

Reply:
I am good with the first option.

Reply:
I've updated the change as suggested